pkgs/applications/video/dmlive/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, openssl
+, mpv
+, ffmpeg
+, nodejs
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dmlive";
+  version = "unstable-2022-08-22";
+
+  src = fetchFromGitHub {
+    owner = "THMonster";
+    repo = pname;
+    rev = "fd4fa1859f05350658db598a50d29f59d22b55a1";
+    hash = "sha256-NVabHLxPHi7hWoztthPmVC5VRKQKglpytuUQOY1Hzrw=";
+  };
+
+  cargoHash = "sha256-TziP7n9Xgi/wHaiF/NI6noMp1iR6vRuAXxvKJwQHbTw=";
+
+  OPENSSL_NO_VENDOR = true;
+
+  nativeBuildInputs = [ pkg-config makeWrapper ];
+  buildInputs = [ openssl ];
```

Comment:
Can you try to fix the darwin build by adding ``Security`` framework for darwin only?